README.md

```diff
@@ -24,17 +24,10 @@ CI runner of YunoHost
 
 ![Screenshot of YunoRunner](./doc/screenshots/screenshot.png)
 
-## Disclaimers / important information
-
-## Limitations
-
-* You need to install [CI_package_check](https://github.com/YunoHost/CI_package_check) using the `install.sh` script before installing YunoRunner
-* When YunoRunner is installed, modify the systemd script to add the path of the script `analyseCI.sh`. The default systemd is configured to `/home/CI_package_check/analyseCI.sh`
-
 ## Documentation and resources
 
 * Upstream app code repository: <https://github.com/YunoHost/yunorunner>
-* YunoHost documentation for this app: <https://yunohost.org/app_yunorunner>
```

Comment:
I did not intend to change this part, but [README-generator](https://github.com/YunoHost/apps/tree/master/tools/README-generator) updated it automatically.